src/services/ApiService.js

```diff
@@ -0,0 +1,33 @@
+import { Config } from "@baltimorecounty/javascript-utilities";
+import axios from "axios";
+
+const { getValue } = Config;
+
+/**
+ * Determines if api is up or not
+ * @returns true if the api is available
+ */
+const GetStatus = () =>
+  axios
+    .get(`${getValue("apiRoot")}/status`)
+    .then(({ status }) => status === 200);
+
+/**
+ * Get News Data from SiteExecutive structured content
+ */
+const GetNews = () =>
+  axios
+    .get("https://structuredcontentdev.bcg.ad.bcgov.us/api/news")
+    .then(({ status, data }) => (status === 200 ? data : []));
+
+/**
+ * Get News Data from SiteExecutive structured content by count
+ */
+const GetMoreNews = (limit, offset) =>
```

Comment:
`GetNews` and `GetMoreNews`, should be the same, we can just pass params to `GetNews`.

```js
GetNews({ page = 0, perPage = 10})
```

or something along those lines

Reply:
will do this in a new branch that is related to Load More.